src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -110,6 +111,18 @@ public EditPersonDescriptorBuilder withPostalCode(String postalCode) {
     }
 
     /**
+     * Sets the {@code PostalCode} of the {@code EditPersonDescriptor} that we are building.
```

Comment:
typo in javadoc, should reflect deadline, not postalcode

Reply:
fixed